identity_resolver/src/resolution/resolver.rs

```diff
@@ -277,6 +279,58 @@ mod iota_handler {
 
       self.attach_handler(IotaDID::METHOD.to_owned(), handler);
     }
+
+    /// Convenience method for attaching multiple handlers responsible for resolving IOTA DIDs
+    /// on multiple networks.
+    ///
+    ///
+    /// # Arguments
+    ///
+    /// * `clients` - A collection of tuples where each tuple contains the name of the network name and its
+    ///   corresponding client.
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
+    /// // Assume `smr_client` and `iota_client` are instances IOTA clients `iota_sdk::client::Client`.
+    /// attach_multiple_iota_handlers(vec![("smr", smr_client), ("iota", iota_client)]);
+    /// ```
+    ///
+    /// # See Also
+    /// - [`attach_handler`](Self::attach_handler).
+    ///
+    /// # Note
+    ///
+    /// - Using `attach_iota_handler` or `attach_handler` for the IOTA method would override all
```

Comment:
```suggestion
    /// - Using `attach_iota_handler` after `attach_handler` (or the other way around) would override all
```

Reply:
This is not accurate, it's using `attach_iota_handler` or `attach_handler` after `attach_multiple_iota_handlers` would overwrite the clients.

Reply:
gotcha, let's go with that